blocked-edges/4.17.17-HostedClusterIsProgressingStuckCondition.yaml

```
@@ -0,0 +1,12 @@
+to: 4.17.17
+from: .*
+name: HostedClusterIsProgressingStuckCondition
+url: https://issues.redhat.com/browse/OCPBUGS-51339
+message: The Hosted Cluster is properly deployed but still incorrectly shows the condition: HostedCluster is deploying, upgrading, or reconfiguring, which is blocking the cluster deployment completion..
```

nitpick: Just a small forgotten dot.

```suggestion
message: The Hosted Cluster is properly deployed but still incorrectly shows the condition: HostedCluster is deploying, upgrading, or reconfiguring, which is blocking the cluster deployment completion.
```